### ISSUE TITLE
Panel: Structure Table Displays "0" as "-"

### ISSUE THREAD

**Issue by neildaniels** [getkirby/kirby#2423]
**Describe the bug**  
When a `structure` field has a `number` sub-field, if the set value is `0`, then the overview table will show `-` instead of `0`.

**To Reproduce**  
1. Have a structure field like:
```yaml
somestructure:
  type: structure
  fields:
    price:
      label: Price
      type: number
      step: 0.01
      min: 0
      placeholder: 0.00
      before: '$'
```
2. Enter in `0` or `0.00`
4. Look at the overview

**Expected behavior**  
I would expect `0` or `0.00` to be displayed. Only if the value was completely empty would I expect to see `-`.

**Screenshots**  
<img width="381" alt="Screen Shot 2020-02-04 at 11 15 43 AM" src="https://user-images.githubusercontent.com/417182/73778640-610de100-4740-11ea-8b1b-99c9a7ec6901.png">
<img width="134" alt="Screen Shot 2020-02-04 at 11 15 36 AM" src="https://user-images.githubusercontent.com/417182/73778644-63703b00-4740-11ea-863d-8ad7bc71cd1b.png">

**Kirby Version**  
`3.3.3`

**Desktop (please complete the following information):**  
 - OS: macOS
 - Browser: Safari
 - Version: 13.0.5 (15608.5.11)

**Comment by afbora** [comment]
Issue on `columnIsEmpty()` method:

````js
{{ displayText(fields[columnName], item[columnName]) || "–" }}
````

When `displayText()` method return `0`, run else condition as `–`

https://github.com/getkirby/kirby/blob/3.3.3/panel/src/components/Forms/Field/StructureField.vue#L115

---

@distantnative What happens if we always return a string? I have tested and working but i'm not sure. Like that: `return value.toString();`

https://github.com/getkirby/kirby/blob/3.3.3/panel/src/components/Forms/Field/StructureField.vue#L416

**Comment by distantnative** [comment]
@afbora I think that should be a good solution. A method like `displayText` can be expected to always return a string.

**Comment by distantnative** [comment]
✅ Will be fixed in 3.3.4
